# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1488815874 18000
#      Mon Mar 06 10:57:54 2017 -0500
# Node ID aaf5ac82979d926f21ebff973c8240187912903c
# Parent  b506b43f999e5f625d8b1f0ea304b8c06090a978
* quit.cc (clean_up_and_exit): Don't declare definition extern.

diff --git a/liboctave/cruft/misc/quit.cc b/liboctave/cruft/misc/quit.cc
--- a/liboctave/cruft/misc/quit.cc
+++ b/liboctave/cruft/misc/quit.cc
@@ -43,17 +43,17 @@ octave_handle_signal (void)
 
   if (octave_interrupt_state > 0)
     {
       octave_interrupt_state = -1;
       octave_throw_interrupt_exception ();
     }
 }
 
-extern OCTAVE_API void
+void
 clean_up_and_exit (int exit_status, bool /* safe_to_return */)
 {
   exit (exit_status);
 }
 
 void
 octave_throw_interrupt_exception (void)
 {
